mod_pbxproj.py

```diff
@@ -743,7 +743,10 @@ def add_run_script(self, target, script=None):
                         skip = True
 
                 if not skip:
-                    t['buildPhases'].add(script_phase.id)
+                    if insert_before_compile is True:
```

Comment:
please replace the `if insert_before_compile is True:` with `if insert_before_compile:` to keep consistency with the rest of the boolean evaluations.